plugins/com.gwtplugins.gwt.eclipse.core/src/com/google/gwt/eclipse/core/modules/AbstractModule.java

```diff
@@ -291,29 +291,29 @@ public String getQualifiedName() {
    * @return the short name for module
    */
   private String getShortName() {
```

Comment:
This method fails when the plugin is scanning classpath for modules, and .gwt.xml files are in jar files.
Returning an empty string should be the right thing to do, as there's some maven related logic which makes sense only for manifests contained in the source, and not in dependencies